src/web/init/docs/Livelits.ml

```diff
@@ -1,154 +1,777 @@
-         if value < 40 then \\\"sad\\\"                       \
-         #\"))))(Secondary((id \
-         90ec4f8e-be0e-4f06-974d-03ba19333337)(content(Whitespace\"\\n\"))))(Secondary((id \
-         1ce4578a-c66e-4e12-9723-fa048d20fa20)(content(Comment\"#                \
-         if value > 70 then \\\"happy\\\"                     \
-         #\"))))(Secondary((id \
-         5486f4b1-477a-4dab-a025-2db31ad83749)(content(Whitespace\"\\n\"))))(Secondary((id \
-         923af167-49f1-4297-a3d6-28f48c14ab37)(content(Comment\"#                \
-         otherwise \\\"neutral\\\"                            \
-         #\"))))(Secondary((id \
-         47960e47-57ce-4528-b034-a99479b99c96)(content(Whitespace\"\\n\"))))(Secondary((id \
-         92427d3b-c08b-4530-b231-f22cfbbf0148)(content(Comment\"# Internal \
-         Model: Int                                           \
-         #\"))))(Secondary((id \
-         6a33cf39-c667-44cf-a4ad-801c161283ba)(content(Whitespace\"\\n\"))))(Tile((id \
-         9b54fd6a-7eed-4e6e-8444-193663808bff)(label(let = in))(mold((out \
-         Exp)(in_(Pat Exp))(nibs(((shape Convex)(sort Exp))((shape(Concave \
-         40))(sort Exp))))))(shards(0 1 2))(children(((Secondary((id \
-         2bda4674-cc71-4dd5-82ab-0cff778a183f)(content(Whitespace\" \
-         \"))))(Tile((id \
-         ecdc9b6c-07da-45cd-9437-daf95f84f7dc)(label(current_mood))(mold((out \
-         Pat)(in_())(nibs(((shape Convex)(sort Pat))((shape Convex)(sort \
-         Pat))))))(shards(0))(children())))(Secondary((id \
-         00a9ab39-2bce-43fd-9327-fe7ddf6a9f49)(content(Whitespace\" \
-         \")))))((Secondary((id \
-         39bf857d-ef27-4ed4-b1df-2f7984b7346c)(content(Whitespace\" \
-         \"))))(Projector((id a73f8254-0425-4968-80c1-71abf84fbc1c)(kind \
-         Livelit)(syntax(Tile((id \
-         fd7c3d96-2016-4143-ab74-536edfaba8fc)(label(\"(\"\")\"))(mold((out \
-         Exp)(in_(Exp))(nibs(((shape Convex)(sort Exp))((shape Convex)(sort \
-         Exp))))))(shards(0 1))(children(((Tile((id \
-         572809ed-59cb-4e1d-9824-a50d257cd1e7)(label(^emotion))(mold((out \
-         Exp)(in_())(nibs(((shape Convex)(sort Exp))((shape Convex)(sort \
-         Exp))))))(shards(0))(children())))(Tile((id \
-         e3c391a3-2c6c-4cd4-ae1b-dc741c9950fd)(label(\"(\"\")\"))(mold((out \
-         Exp)(in_(Exp))(nibs(((shape(Concave 23))(sort Exp))((shape \
-         Convex)(sort Exp))))))(shards(0 1))(children(((Tile((id \
-         7e7a62e9-51a3-4bbb-9555-54e8b2f2c869)(label(0))(mold((out \
-         Exp)(in_())(nibs(((shape Convex)(sort Exp))((shape Convex)(sort \
-         Exp))))))(shards(0))(children()))))))))))))))(model\"()\")))(Secondary((id \
-         44ddd530-82a8-4184-800a-21652348cfe7)(content(Whitespace\" \
-         \")))))))))(Grout((id 3534f38b-9cab-4478-9cde-bd0d16f2fae0)(shape \
-         Convex))))))(ancestors())))(caret Outer))";
+      segment =
```

Comment:
@disconcision I'm conflicted between storing this as the sexp or just the ml code. The ml code doesn't have to parse but the formatted code is so much longer. Do you have any thoughts?

Reply:
how would export work to generate the ml code?

Reply:
Basically this code https://github.com/hazelgrove/hazel/pull/1919/files#diff-bf0419f9509267c72a8bef1b8106779b0711842cd703b69d97e330e612c995d0R73-R83